### ISSUE TITLE
Can't sign transactions with social login wallet

### ISSUE THREAD

**Issue by akarys92** [Web3Auth/Web3Auth#73]
I've integrated web3auth into my Solana web app. I'm able to connect with a social login on the home page of the modal and the Torus Wallet under "External Wallet" via my google login. I'm able to sign transactions with Torus which seems to work great. 

However, when I connect using the social login on the home page of the modal, it successfully connects but when I go to sign a transaction with the wallet using `signTransaction` I get the error: `Error: Transaction fee payer required`. 

Since the Torus integration works, I think I am specifying a fee payer correctly, something isn't getting configured right now the web3auth side.

**Comment by himanshuchawla009** [comment]
can you plz add a code snippet to show how you are signing? @akarys92

**Comment by akarys92** [comment]
This is the test method I'm using: 

`const transactionTest2 = async () => {
        const lamportPayment = 999; 
        const program = new Program(idl, programID, provider); 
        const recipient = new web3.PublicKey("EjVxdTp7NXqUtqSACXeMojnTUwkFhr1wd5rxmDSLrstd"); 
        console.log("Sending " + lamportPayment + " lamports...");
        try {
            
            let transaction = new Transaction();

            transaction.add(
                SystemProgram.transfer({
                    fromPubkey: program.provider.wallet.publicKey,
                    toPubkey: recipient,
                    lamports: lamportPayment
                })
            );

            const connection = new Connection("https://api.devnet.solana.com");

            transaction.feePayer = await provider.wallet.publicKey; 

            let blockhashObj = await connection.getRecentBlockhash();

            transaction.recentBlockhash = await blockhashObj.blockhash;


            let signed = await provider.wallet.signTransaction(transaction);
            
            let signature = await connection.sendRawTransaction(signed.serialize());
            await connection.confirmTransaction(signature);
            console.log("Signature: ", signature);
            console.log("Successfully sent funds!");
        }
        catch(e) {
            console.log("Error sending funds: \n" + e);
        }
    }`

**Comment by akarys92** [comment]
Also in case it's helpful here's how I set it up: 

```
const solanaChainConfig = {
    chainNamespace: CHAIN_NAMESPACES.SOLANA,
    rpcTarget: "https://api.devnet.solana.com",
    blockExplorer: "https://explorer.solana.com?cluster=devnet",
    chainId: "0x3",
    displayName: "devnet",
    ticker: "SOL",
    tickerName: "solana",
};

// Init web3auth when first loading component
    const init = async () => {
      let newWeb3Auth = new Web3Auth({
          chainConfig: solanaChainConfig,
          clientId: web3AuthClientID
      });
      await newWeb3Auth.initModal();
      if(newWeb3Auth.status === "connected") {
          setStatus(STATES.CONNECTED);
      }
      else {
          setStatus(STATES.DISCONNECTED); 
      }
      subscribeAuthEvents(newWeb3Auth);
      setWeb3Auth(newWeb3Auth);
  }

// Open the web3auth connection modal
    const connectModal = async () => {
        const tempProvider = await web3auth.connect();
        const solWeb3 = new SolanaWallet(web3auth.provider);

        const pubKey = await solWeb3.requestAccounts();
        const connection = new Connection(network, opts.preflightCommitment); 
        
        solWeb3.publicKey = new PublicKey(pubKey[0]);  
        const provider = new Provider(
          connection, solWeb3, opts.preflightCommitment, 
        ); 

        const userInfo = await web3auth.getUserInfo();

        setUserInfo(userInfo); 
        setProvider(provider);
        setSolweb3(solWeb3);
        setPubkey(pubKey);
        setIsConnected(true);
    }
```

**Comment by akarys92** [comment]
Hey @himanshuchawla009 any update on this? Let me know if there’s more info I can provide to help!

**Comment by himanshuchawla009** [comment]
resolved in v0.5.1 @akarys92

**Comment by himanshuchawla009** [comment]
resolved in v0.5.1 @akarys92 , closing it , feel free to reopen if you still face any issue.

PS: Please use latest version of @solana/web3.
